IHP/HtmlSupport/Parser.hs

```diff
@@ -194,7 +194,9 @@ hsxSplicedValue = do
     pure (ExpressionValue haskellExpression)
 
 hsxClosingElement name = do
-    _ <- string ("</" <> name <> ">")
+    _ <- string ("</" <> name)
+    space
+    char ('>')
     pure ()
```

Comment:
Can't we add a `<?> ("</" <> name <> ">")` here to improve the error message?
Like this:

```
hsxClosingElement name = (hsxClosingElement' name) <?> ("</" <> name <> ">")
    where
        hsxClosingElement' name = do
    _ <- string ("</" <> name <> ">")
    _ <- string ("</" <> name)
    space
    char ('>')
    pure ()
```